apm-agent-plugins/apm-jms-plugin/src/test/java/co/elastic/apm/agent/jms/JmsInstrumentationIT.java

```diff
@@ -475,12 +490,14 @@ private void verifySendReceiveOnNonTracedThread(String destinationName, TextMess
             assertThat(receiveTransaction.getNameAsString()).endsWith(destinationName);
             assertThat(receiveTransaction.getTraceContext().getTraceId()).isEqualTo(currentTraceId);
             assertThat(receiveTransaction.getTraceContext().getParentId()).isEqualTo(sendInitialMessageSpan.getTraceContext().getId());
+            assertThat(receiveTransaction.getType()).isEqualTo(MESSAGING_TYPE);
             if (isQueue) {
                 assertThat(receiveTransaction.getContext().getMessage().getQueueName()).isEqualTo(destinationName);
             } else {
                 assertThat(receiveTransaction.getContext().getMessage().getTopicName()).isEqualTo(destinationName);
             }
             assertThat(receiveTransaction.getContext().getMessage().getBody()).isEqualTo(message.getText());
+            assertThat(receiveTransaction.getContext().getMessage().getAge()).isGreaterThan(0);
```

Comment:
same as previous, why not greater than or equal to zero ?

Reply:
Since there are no clock skews within the same machine and since the time is not expected to be 0 - I assume that.
But semantically you are correct